pkgs/development/python-modules/py-mini-racer/default.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+}:
+
+buildPythonPackage rec {
+  pname = "py-mini-racer";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "sqreen";
+    repo = "PyMiniRacer";
```

Comment:
This repository is deprecated. The successor is https://github.com/bpcreech/PyMiniRacer.